README.rst

```diff
@@ -20,8 +20,13 @@ Silicone
 .. sec-begin-long-description
 .. sec-begin-index
 
-Silicone is a Python package which can be used to automatically fill in detail in emissions scenarios.
-For example, adding emissions of aerosol precurors and methane based on carbon dioxide and nitrous oxide emissions pathways or adding regional emissions based on a global emissions pathway.
+Silicone is a Python package which can be used to infer predictions about certain types
+of emissions from other predicted emissions. It is intended to ensure more integrated
+assessment models (IAMs) have predictions of all climate-relevant emissions.
+It does this by comparing the incomplete emissions set to completed data.
```

Comment:
```suggestion
It does this by comparing the incomplete emissions set to complete data. It uses the relationships within the complete data to make informed infilling estimates of otherwise missing emissions timeseries.
```

Yuck suggestion from me but hopefully inspires something better